### DISCUSSION TITLE
Difference between common attributes and attributes in node mask of gnn explainer

### DISCUSSION THREAD

**Discussion by bojack-horseman91** [pyg-team/pytorch_geometric#9292] [Q&A]
I want to get a intuiative understanding of what the difference is. I read the doc where it said about that one masks the attributes and the other masks all feature accross all nodes I want to understand how we should interpret them differently?Like if I have a common attribute explanation and a attribute explanation how should we interpret them differently

**Comment by rusty1s** [comment]
Imagine you have a feature called "age" for all users. "common attributes" would refer to the global importance of the "age" among all users that are part of the subgraph (i.e., if you remove this feature from the dataset, which impact would that have). "attributes" assigns an importance score to the "age" column for all users, e.g., the "age" of the seed entity might be more important than the "age" of neighbors.
